src/screens/AirdropFinished/AirdropFinished.tsx

```diff
@@ -0,0 +1,76 @@
+import { RouteProp } from '@react-navigation/native';
+import React, { FC } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+
+import {
+  ScreenTemplate,
+  Header,
+  AirdropWalletBalanceCard,
+  SocialShareFacebookButton,
+  SocialShareTwitterButton,
+} from 'app/components';
+import { Route, MainCardStackNavigatorParams } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+type Props = {
+  route: RouteProp<MainCardStackNavigatorParams, Route.AirdropFinished>;
+};
+
+export const AirdropFinished: FC<Props> = ({ route }: Props) => (
+  <ScreenTemplate header={<Header isBackArrow title={i18n.airdrop.title} />}>
+    <Text style={styles.subtitle}>{i18n.airdrop.finished.subtitle}</Text>
+    <AirdropWalletBalanceCard walletDetails={{ balance: route.params.balance, name: route.params.name }} />
+    <View style={styles.stayTunedContainer}>
+      <Text style={styles.firstLine}>
+        {i18n.formatString(i18n.airdrop.finished.stayTuned, {
+          period: 'Q1 2021',
```

Comment:
I'm wondering if it wouldn't be a good idea to keep in somewhere else, like in envs. I mean, I'm not sure, but it doesn't sound like a good design to must change a codebase every time when a new period starts. 

What do you think, @jellyfish-tom , @KedziaPawel ?

Reply:
we have lots such values, currently they are kept in `consts`. I can move it there, which probably will look better, but won't change the need to update codebase with each new period.

Reply:
true. 

maybe we should ask our backend colleagues for exposing an endpoint for that or to include this information in the responses? 🤔

Reply:
might be a good idea

Reply:
please move to const, for this airdrop we established that backend won't send it, later in the future we will create a more generic advanced solution with an airdrop configurator, etc.

Reply:
done